misc/packaging/gen_acrn_deb.py

```diff
@@ -70,6 +70,7 @@ def create_acrn_deb(board, scenario, version, build_dir):
 
     listcontrol=['Package: acrn-hypervisor\n',
                 'version: %s \n'% version,
+		'Depends: libcjson-dev\n',
```

Comment:
This is not correct. The `-dev` package would be a `build-depends` but you do not need it on the target system (which is where the Debian package will be installed). All you need there is the `libcjson1` package here.

I have not played with `build-depends`, and we don't have any of those listed in the current packaging. I would suggest we do not address it now and defer that to a different PR that would be an enhancement to the Debian packaging in general.

So for this PR: 
```suggestion
		'Depends: libcjson1\n',
```

Reply:
**Note**: you also need to update the GSG and [this section](https://projectacrn.github.io/latest/getting-started/getting-started.html?#prepare-the-development-computer) in particular to add the `libcjson-dev` package to the list of build tools.

Reply:
Got it, libcjson1 is fine, I will update this PR, BTW, Nanlin will ask other to update GSG about libcjson-dev. Thanks!

Reply:
> Got it, libcjson1 is fine, I will update this PR, BTW, Nanlin will ask other to update GSG about libcjson-dev. Thanks!

It's literally a one-line patch to update the GSG, can you add this to your PR? (Instead of asking someone else to send yet another PR)

```
$ git diff
diff --git a/doc/getting-started/getting-started.rst b/doc/getting-started/getting-started.rst
index 0f57ab31f..51f11567f 100644
--- a/doc/getting-started/getting-started.rst
+++ b/doc/getting-started/getting-started.rst
@@ -126,6 +126,7 @@ To set up the ACRN build environment on the development computer:
            e2fslibs-dev \
            pkg-config \
            libnuma-dev \
+           libcjson-dev \
            liblz4-tool \
            flex \
            bison \
```

Reply:
Thanks! Will add patch to update GSG in this PR.